extension/chrome/settings/setup/setup-render.ts

```diff
@@ -107,11 +107,12 @@ export class SetupRenderModule {
       return await Settings.promptToRetry(e, Lang.setup.failedToCheckIfAcctUsesEncryption, () => this.renderSetupDialog(),
         Lang.general.contactIfNeedAssistance(this.view.isFesUsed()));
     }
+    const scopes = await AcctStore.getScopes(this.view.acctEmail);
     if (keyserverRes.pubkeys.length) {
       if (!this.view.clientConfiguration.canBackupKeys()) {
         // they already have a key recorded on attester, but no backups allowed on the domain. They should enter their prv manually
         this.displayBlock('step_2b_manual_enter');
-      } else if (this.view.storage!.email_provider === 'gmail' && this.view.scopes!.modify) {
+      } else if (this.view.storage!.email_provider === 'gmail' && scopes.modify) {
```

Comment:
If I understand correctly, we decided to not check `scopes.modify` and this was implemented by your fix #4611

I looked at the code and see that `GoogleAuth.newOpenidAuthPopup` is never used and can be removed, hence `GoogleAuth.defaultScopes('openid')` is never used and this option can be removed, therefore `modify` should be present for all (at least new) setups, right?

Reply:
Yeah, I agree